extensions/interactions/base.py

```diff
@@ -89,6 +89,11 @@ class BaseInteraction(object):
     # containing this interaction. These should correspond to names of files in
     # feconf.DEPENDENCIES_TEMPLATES_DIR. Overridden in subclasses.
     _dependency_ids = []
+    # Additional JS library dependencies that should be loaded in pages
```

Comment:
@seanlip OK, then let's close this PR. I will work on `$injector` part when, in my next GSoC milestone (which will enable code classifier in Oppia, so that we can start end to end testing in development mode). Is that fine?